Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs

```diff
@@ -1096,27 +1090,12 @@ private void DropController()
 
         private void DestroyRig()
         {
-            if (boundsOverride == null)
+            // If we have previously logged an initial bounds size,
+            // reset the boundsOverride BoxCollider to the initial size.
+            // This is because the CalculateBoxPadding
+            if (initialBoundsOverrideSize.HasValue)
             {
-                Destroy(TargetBounds);
```

Comment:
I could see us wanting to track `TargetBounds` and `NearInteractionGrabbable` if this script adds them at runtime, so we properly clean ourselves up when it's destroyed and not leave stray components behind. We could track if _we're_ the one adding them as opposed to the developer adding them manually. Or do you think that's not as important in this case?

Reply:
Stewing of this a bit, I want to get rid of us automatically setting NearInteractionGrabbable (ObjectManipulator doesn't even do this), and want to make an in editor tool for generating the bounds control bounding box rather than handling it all implicitly in code. 

Baiscally, the bounds override will always be set and adjusted appropriately, rather than a wishy-washy ability to set it or to let it generate automatically.